aws/src/main/java/org/apache/iceberg/aws/glue/GlueTableOperations.java

```diff
@@ -198,6 +208,17 @@ protected void doCommit(TableMetadata base, TableMetadata metadata) {
     }
   }
 
+  private FileIO initializeFileIO(Map<String, String> properties) {
```

Comment:
I think we need to make initializeFileIO static method to be used between this class and GlueCatalog.